pkgs/servers/headphones/default.nix

```diff
@@ -0,0 +1,33 @@
+{ stdenv, fetchFromGitHub, python2, makeWrapper }:
+
+python2.pkgs.buildPythonApplication rec {
+  name = "headphones-${version}";
+  version = "0.5.19";
+
+  src = fetchFromGitHub {
+    owner = "rembo10";
+    repo = "headphones";
+    rev = "v${version}";
+    sha256 = "0z39gyan3ksdhnjxxs7byamrzmrk8cn15g300iqigzvgidff1lq0";
+  };
+
+  dontBuild = true;
+  doCheck = false;
+
+  nativeBuildInputs = [ makeWrapper ];
+  buildInputs = [ python2 ];
+
+  installPhase = ''
+    mkdir -p $out/bin
+    cp -R {data,headphones,lib,Headphones.py} $out/
+
+    makeWrapper $out/Headphones.py $out/bin/headphones
```

Comment:
Isn't this almost the same as just copying `Headphones.py` to `bin/headphones`? Except without polluting top-level `$out` and in a way that is convenient for `patchShebangs`.

Reply:
No - that doesn't seem to work because it breaks the imports. Unless I'm doing something wrong? I just changed the makeWrapper line to `cp $out/Headphones.py $out/bin/headphones`

Reply:
Because the imports are relative to where the main script is called - so it's no longer the case if the main file is being copied to and run from another location

Reply:
Ahh so `Headphones.py` needs to be at that location. That was my assumption at first.

Reply:
Hm. As the `lib/` layout is not for importing this as a library package anyway, in such cases I do something like `$out/lib/headphones`, copy the entire source-directory there, then wrap the scripts from there.